pkg/daemon/ceph/client/filesystem.go

```diff
@@ -173,6 +174,11 @@ func IsMultiFSEnabled() bool {
 	}
 	return false
 }
+func SetPools(context *clusterd.Context, cephVersion cephver.CephVersion, clusterName, fsName string, metdataPool []*model.Pool, dataPool *model.Pool) error {
+	args := []string{"osd", "pool", "set", fsName, "size"}
```

Comment:
I think you're missing an argument, which is the desired size.

Reply:
Yes, and it'll be the DataPool and the MetadataPool Replicated size, I'm having trouble with the data type mismatch.
`size (metadataPool)`  throws an error that it cannot convert to int.
Wondering what dataType would it be.

Reply:
data type of? The CLI should be:

```go
args := []string{"osd", "pool", "set", fsName, "size", replicaCount}
```

`replicaCount` is a string.